src/Inkluzitron/Services/PointsService.cs

```diff
@@ -17,7 +18,7 @@
 
 namespace Inkluzitron.Services
 {
-    public class PointsService
+    public class PointsService : IReactionHandler
```

Comment:
Moc se mi nelíbí, že je to ReactionHandler a současně singleton. Nezaregistruje se to 2x? Nemůže to pak dělat problémy?

Reply:
This: https://github.com/Inkluzitron/Inkluzitron/blob/vote/src/Inkluzitron/Program.cs#L86
Vs. https://github.com/Inkluzitron/Inkluzitron/blob/vote/src/Inkluzitron/Program.cs#L116

Reply:
Máš pravdu, nejspíš by singletony žily ve dvou instancích. Přidal jsem extension metodu `IServiceCollection.RegisterAs`, která by tohle měla řešit.